app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -1326,11 +1330,7 @@ public static void bgReadingInsertFromInt(int value, long timestamp, long margin
                 if (readingNearTimeStamp(bgr.timestamp, margin) == null) {
                     bgr.save();
                     bgr.find_slope();
-                    if (do_notification) {
```

Comment:
What happened to this part? Why is it removed?

Reply:
This is moved to bgr.postProcess(!do_notification, true); see other insert functions as well

Reply:
This is moved to bgr.postProcess(!do_notification, true); see other insert functions as well